
## Genomics Data

The Biomass Association Panel planted in Seasons 4 and 6 was described by @brenton2016. These genotypes have been sequenced and the sequence data and SNPs are provided as an archive on Dryad. A copy of these data is available on the CyVerse Data Store https://datacommons.cyverse.org/browse/iplant/home/shared/terraref.

The genomics dataset includes raw and derived sorghum genome sequencing data from the TERRA-REF project. Raw data includes DNA sequence files in compressed FASTQ format. Derived data is available for whole-genome resequencing and genotyping-by-sequencing.

These data are archived in the same filesystem as the sensor data, and available at the Globus ncsa#terra-public endpoint under the `genomics/` directory. A copy is available in the CyVerse data store. 

At both Globus and CyVerse, the data is organized in the following directory structure:

```
|-data
|  |-genomics
|  |  |-raw_data
|  |  |  |-bap
|  |  |  |  |-resequencing
|  |  |  |-ril
|  |  |  |  |-gbs
|  |  |-derived_data
|  |  |  |-bap
|  |  |  |  |-resequencing
|  |  |  |  |  |-danforth_center
|  |  |  |-ril
|  |  |  |  |-gbs
|  |  |  |  |  |-kansas_state
```

### Germplasm

 A list of cultivars and the experiments in which they are planted are listed in the file `metadata/germplasm.csv`.

 The `germplasm.csv` file contains the following fields:

* **germplasmName** Genotype identifier used by TERRA REF; typically equal to Genesys Accession Number
* **germplasmPUI** UUID from [Genesys](https://www.genesys-pgr.org)
* **season_4** A boolean which is _TRUE_ if the accession was planted in Season 4 experiments.
* **season_6** A boolean value which is _TRUE_ if the accession was planted in Season 6 experiments.

For user convenience, we also provide this information and additional metadata in a format that follows the Breeder's API (BrAPI) specification v1.3 @Selby2019 in files named `season_[n]_germplasm.json` and in the `terraref_sorghum_bap_samples.txt` table alongside raw sequencing data (described below).

### Whole-genome resequencing data for the sorghum Bioenergy Association Panel (BAP)

#### Raw data

There are 384 available samples. Each sample is stored as a pair of files in bzip2 FASTQ format with names ending in `*_R1.fastq.bz2` and `*_R2.fastq.bz2`. 
R1 and R2 in these names represent read pairs one and two.
Each file in a pair has the same number of DNA sequences that line up and are read from opposite ends of the same sequenced molecule.

A table of sample information (`terraref_sorghum_bap_samples.txt`) can be found in the
[raw data directory `genomics/raw_data/bap/resequencing`](https://datacommons.cyverse.org/browse/iplant/home/shared/terraref/genomics/raw_data/bap/resequencing).

Each table contains the following fields:

* **identifier**: The unique genotype identifier that will be used in the derived data outputs
* **accession**: Genotype identifier used by TERRA REF
* **taxid**: Taxonomy identifier from the [NCBI taxonomy database](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=4558)
* **organism common name**: Genbank common name from the [NCBI taxonomy database](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=4558)
* **subspecific genetic lineage rank**: Cultivar
* **subspecific genetic lineage name**: Unique germplasm identifier
* **ploidy**: Source, [Phytozome](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor)
* **number of replicons**: Number of chromosomes, source [Phytozome](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor)
* **estimated size**: Estimated genome size, source [Phytozome](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor)
* **source material identifiers**: UUID from [Genesys](https://www.genesys-pgr.org)
* **sample name**: Internal sample identifier
* **raw file 1**: File name for read-pair 1
* **raw file 2**: Filename for read-pair 2

#### Derived data

Data derived from analysis of the raw resequencing data at the Danforth Center (version1)
are available as gzipped, genotyped variant call format (gVCF) files and the final
combined hapmap file in the
 [derived data directory,  `genomics/derived_data/bap/resequencing/danforth_center/version1`](https://datacommons.cyverse.org/browse/iplant/home/shared/terraref/genomics/derived_data/bap/resequencing/danforth_center/version1).

### Genotyping-by-sequencing (GBS) data for sorghum recombinant inbred lines (RIL)

#### Raw data

Raw data are in gzip FASTQ format. There are 768 available samples.

A key file (`Key_ril_terra`) with sample information can be found in the
[raw data directory, `genomics/raw_data/ril/gbs`](https://datacommons.cyverse.org/browse/iplant/home/shared/terraref/genomics/raw_data/ril/gbs).

#### Derived data

Combined genotype calls are available in VCF format in the
[derived data directory,\newline `genomics/derived_data/ril/gbs/kansas_state/version1`](https://datacommons.cyverse.org/browse/iplant/home/shared/terraref/genomics/derived_data/ril/gbs/kansas_state/version1).
